packages/abi-coder/test/fixtures/forc-projects/exhaustive-examples/src/main.sw

```diff
@@ -339,4 +342,8 @@ impl MyContract for Contract {
     fn simple_vector(arg: Vec<u8>) -> Vec<u8> {
         arg
     }
+    #[payable]
+    fn send_typed_message_bool(recipient: b256, msg_data: bool, coins: u64) {
+        send_typed_message(recipient, msg_data, coins);
+    }
```

Comment:
Should this have a related test?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3016/commits/9aeccc14367f8ae379de6d85d607451e2e34d7c5